docs/administrator-guide/configuration/enterprise-edition/workers/README.md

```diff
@@ -22,19 +22,20 @@ kestra:
 ```
 
 ### `kestra.ee.java-security.forbidden-paths`
-This is the filepath on the system that the Kestra Worker will be forbidden to read or write at all times. This can be useful to protect Kestra configuration files for example.
+This is a list of path on the file system that the Kestra Worker will be forbidden to read or write to. This can be useful to protect Kestra configuration files for example.
```

Comment:
```suggestion
This is a list of paths on the file system that the Kestra Worker will be forbidden to read or write to. This can be useful to protect Kestra configuration files for example.
```